rust/operator-binary/src/main.rs

```diff
@@ -36,13 +36,21 @@ pub mod built_info {
 #[clap(about, author)]
 struct Opts {
     #[clap(subcommand)]
-    cmd: Command<OpaRun>,
+    cmd: Cmd,
+}
+
+#[derive(Parser)]
+enum Cmd {
+    #[clap(flatten)]
+    Common(Command<OpaRun>),
 }
 
 #[derive(clap::Parser)]
 struct OpaRun {
     #[clap(long, env)]
     opa_bundle_builder_clusterrole: String,
+    #[clap(long, env)]
+    operator_image: String,
```

Comment:
If we're moving the envvar into templating.. should this be moved into `ProductOperatorRun`? not sure

Reply:
Yes, it would be added to https://github.com/stackabletech/operator-rs/blob/f9a13a41b5637cb87b270600790b7b8604371d0a/src/cli.rs#L190-L200. properties.yaml (product-config) would not be affected.

Reply:
I have created a branch and added the option, but I'm unsure if it should have a default empty value, and what the doc-comment description should be.

https://github.com/stackabletech/operator-rs/compare/main...product-operator-run-operator-image

Reply:
My suggestion:
```
    /// The full name of the operator image. This env var is set by the helm-charts and should
    /// therefore always be present.
    #[arg(long, env)]
    pub operator_image: String,
}
```

Reply:
Cool, updated on the branch. Here's the PR: https://github.com/stackabletech/operator-rs/pull/702

Reply:
That PR was closed as it was deemed too premature.